credentials/test/gen-test-attestation-certs.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+#
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Script that can be used to generate CHIP device attestation certificates
+# for testing purposes.
+# The script expects the path to the chip-cert tool binary as an input argument.
+#
+# Usage example when the script is run from the CHIP SDK root directory:
+#     ./credentials/test/gen-test-attestation-certs.sh ./out/debug/standalone/chip-cert
+#
+# The result will be stored in:
+#     credentials/test/attestation
+#
+# If the intention is to re-generate a new set of attestation certificates that
+# replace the already present ones then it is recommended to remove current certificates:
+#     rm credentials/test/attestation/*
+#
+
+set -e
+
+here=${0%/*}
+
+dest_dir="$here/attestation"
+
+mkdir -p "$dest_dir"
+
+if [ $# == 1 ]; then
+    chip_cert_tool=$1
+else
+    echo "Error: Please specify exactly one input argument; the path to the chip-cert tool binary"
+    exit
+fi
+
+cert_valid_from="2021-06-28 14:23:43"
+cert_lifetime=7305
+
+paa_vids=(FFF1 FFF2)
+pai_pids=(ABC0 ABC1)
```

Comment:
Suggest using 0000 and 0001 for these, or 0x8000, 0x8001

Reply:
done